libsolidity/codegen/ir/IRGenerator.h

```diff
@@ -87,10 +87,10 @@ class IRGenerator
 	/// Generates code that assigns the initial value of the respective type.
 	std::string generateInitialAssignment(VariableDeclaration const& _varDecl);
 
-	/// Generates implicit constructors for all contracts in the inheritance hierarchy of
+	/// Generates constructors for all contracts in the inheritance hierarchy of
 	/// @a _contract
-	/// If there are user defined constructors, their body will be included in implicit constructors body.
-	void generateImplicitConstructors(ContractDefinition const& _contract);
+	/// If there are user defined constructors, their body will be included in the implicit constructors body.
```

Comment:
```suggestion
	/// If there are user defined constructors, their body will be included in the implicit constructors' body.
```
Optional.

Reply:
Oh. It's plural. Then I think it should be:
```c++
/// If there are user defined constructors, their bodies will be included in the implicit constructors' bodies.
```

Reply:
If it were plural it would be `bodies` though :P

Reply:
Yes, that's what I'm suggesting above :)

Reply:
Oh. Well. I mean, it could be either I suppose?

Reply:
I think it's sounds a bit weird with `body`. But I also didn't even notice that when I read the sentence first time so the meaning is clear anyway :)